### PR TITLE
Added apple

### PR DESCRIPTION
**Pull request by andrewdeanphillips**

### CODE REVIEW

**Review by Happyman31 — DISMISSED** [review]
LGTM.

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias"]
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "apple"]
```
Comment (lach-j) [inline Foods.yaml]: An apple a day keeps the doctor away :apple:

Comment (CMCD1998) [inline Foods.yaml]: what a shit food choice get gud scrub

**Review by CMCD1998 — DISMISSED** [review]
what a shit food

**Review by CMCD1998 — DISMISSED** [review]
WOW

**Review by CMCD1998 — APPROVED** [review]
you hair is terrible

**Pull request merged**